react-components/src/apps/quickLaunch/QuickLaunchListing.js

```diff
@@ -236,12 +239,11 @@ function ListQuickLaunches(props) {
                                             }
                                         />
                                         <ActionsPopper
```

Comment:
I think the heart of the issue is here, which was kinda fun to spot!

This is a popover that will open only if `anchorEl` is set and anchor to whatever the current `anchorEl` is, but this is within the `map` function.  So whenever you click a quick launch, the `anchorEl` gets set to that element, then N number of popovers all render at the same time, all anchored to the same element which you just clicked. The last element is the one you see on top and thus the only one you can interact with.

My guess is if you find a way to move the `ActionsPopper` rendering out of `map`, it should work.

Reply:
I think you're right! Good catch! 🔍